src/Command.ts

```diff
@@ -352,4 +354,26 @@ export default abstract class Command {
       payload[o] = unknownOptions[o];
     });
   }
+
+  private loadValuesFromAccessToken(args: CommandArgs) {
+    if (!args || !args.options || args.options.length === 0 || !auth.service.accessTokens[auth.defaultResource]) {
+      return;
+    }
+
+    const token = auth.service.accessTokens[auth.defaultResource].accessToken;
+    const optionNames: string[] = Object.getOwnPropertyNames(args.options);
+    optionNames.forEach(option => {
+      const value = args.options[option];
+      if (!value || typeof value !== 'string') {
+        return;  
+      }
+
+      if (value.toLowerCase().includes('@meid')) {
+        args.options[option] = accessToken.getUserIdFromAccessToken(token);
+      }
+      if (value.toLowerCase().includes('@meusername')) {
+        args.options[option] = accessToken.getUserNameFromAccessToken(token);
+      }
```

Comment:
I suppose we could do `toLowerCase()` once before the conditions and then just reuse the variable. Now it is also ok 😉. I would say this could be just a 'cosmetic improvement' 😉

Reply:
Hi,
did this change.

Thank you